### PR TITLE
Devl/new functionality

### PR DESCRIPTION
**Pull request by kelvin-onlab**
All the driver changes and test on I made is already in this branch, the rest of the test modification I would do is when I refactor it in the new gerrit repo~

### CODE REVIEW

File: TestON/drivers/common/cli/emulator/mininetclidriver.py [hunk 1]
```diff
@@ -61,6 +62,22 @@ def connect( self, **connectargs ):
                 vars( self )[ key ] = connectargs[ key ]
 
             self.name = self.options[ 'name' ]
+
+            try:
+                if os.getenv( str( self.ip_address ) ) != None:
```
Comment (jhall11) [inline TestON/drivers/common/cli/emulator/mininetclidriver.py]: What are you trying to do here?

File: TestON/drivers/common/cli/onosdriver.py [hunk 2]
```diff
@@ -2259,6 +2274,64 @@ def onosErrorLog(self, nodeIp):
             main.log.info(msg3)
         main.log.info("===============================================================\n")
 
+<<<<<<< HEAD
```
Comment (jhall11) [inline TestON/drivers/common/cli/onosdriver.py]: Looks like you have an unresolved merge conflict here

File: TestON/tests/FUNCintent/FUNCintent.py [hunk 3]
```diff
@@ -150,7 +160,7 @@ def CASE10( self, main ):
                                  onpass="Successfully installed ONOS package",
                                  onfail="Failed to install ONOS package" )
 
-        time.sleep( 20 )
+        time.sleep( 60 )
```
Comment (jhall11) [inline TestON/tests/FUNCintent/FUNCintent.py]: can you make this a parameter and document what this is?

Comment (hakrishn) [inline TestON/tests/FUNCintent/FUNCintent.py]: I think we need to do this for all time.sleep() statements where ever we use in test scripts. Params file will include them with default values assigned (or working values in our setups), that way an end user would be able change them depending on their testbeds (example: someone running ONOS in AWS may have slow connection)

File: TestON/drivers/common/cli/emulator/mininetclidriver.py [hunk 4]
```diff
@@ -61,6 +62,22 @@ def connect( self, **connectargs ):
                 vars( self )[ key ] = connectargs[ key ]
 
             self.name = self.options[ 'name' ]
+
+            try:
```
Comment (kelvin-onlab) [inline TestON/drivers/common/cli/emulator/mininetclidriver.py]: Sorry i forgot to document  it, ill push it again after the planning with complete documentation. but it  basically just grabs the os environment variables and set it as the ip address, its not completely structured yet but it works for now. The normal passing of the ip address is still working by the way. No change necessary in topo file for now.

File: TestON/drivers/common/cli/onosdriver.py (hunk 2 above)
Comment (kelvin-onlab) [inline TestON/drivers/common/cli/onosdriver.py]: Thanks!

**Comment by kelvin-onlab** [comment]
Fixing a lot of change this last commit was a mess. closing it for now

**Pull request closed without merging**